src/main/kotlin/com/project/geo/service/impl/StreetGeometryServiceImpl.kt

```diff
@@ -0,0 +1,57 @@
+package com.project.geo.service.impl
+
+import com.mapbox.geojson.LineString
+import com.mapbox.geojson.Point
+import com.project.geo.dto.StreetResponse
+import com.project.geo.service.StreetGeometryService
+import com.project.geo.utils.deserializeByClass
+import org.springframework.http.HttpStatusCode
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestClient
+import org.springframework.web.client.body
+
+@Service
+class StreetGeometryServiceImpl(
+    private val overpassClient: RestClient
+) : StreetGeometryService {
```

Comment:
move logic for making requests to the client component, logic for converting to geojson leave here